tests/test_stats.py

```
@@ -91,3 +93,85 @@ def test_run_stats_nested_provider_dirs(tmp_path, capsys):
+
+    monkeypatch.setattr("polylogue.cli.status.DEFAULT_OUTPUT_ROOTS", [render_dir.parent, codex_dir.parent])
+    args = Namespace(dir=None, json=True, since=None, until=None)
+    env = CommandEnv(ui=UI(plain=True))
+    run_stats_cli(args, env)
+    payload = json.loads(capsys.readouterr().out)
+    assert payload["totals"]["files"] == 2
+    assert set(payload["providers"].keys()) == {"chatgpt", "codex"}
+    assert set(payload["directories"]) == {str(render_dir.parent), str(codex_dir.parent)}
+
+
+def test_run_stats_ignore_legacy(monkeypatch, tmp_path, capsys):
```

Ruff (0.15.9)

[warning] Unused function argument: `monkeypatch`

(ARG001)